csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -11,6 +11,7 @@
 
 namespace Ice
 {
+    // TODO: rename Disp to Dispatcher or Servant and fix its signature
```

Comment:
Why the possibility of "Servant"? To me it's "Dispatcher", a servant is a class that implements IObject and provides a Dispatcher method.

Reply:
Comment fixed.